openeogeotrellis/logs.py

```diff
@@ -34,6 +38,15 @@ def _elasticsearch_logs(job_id: str, search_after: Optional[list]) -> Iterable[d
                             "filter": [
                                 {"term": {"job_id": job_id}},
                                 {"terms": {"tags": ES_TAGS}},
+                                {
+                                    "range": {
+                                        "@timestamp": {
+                                            "format": "strict_date_optional_time",
```

Comment:
`date_time` or `strict_date_time` might be a better fit (https://www.elastic.co/guide/en/elasticsearch/reference/7.17/mapping-date-format.html).

Reply:
OK, I changed it to strict_date_time.

FYI that code has moved a little bit, because I also made `create_time` an optional parameter of the function.
See: 
https://github.com/Open-EO/openeo-geopyspark-driver/blob/issue307_improve-error-message-when-log-collection-fails/openeogeotrellis/logs.py#L78-L96